lib/github/calendar.rb

```diff
@@ -0,0 +1,184 @@
+require 'open-uri'
+require 'nokogiri'
+require 'string-utility'
+
+module GitHub
+  module Calendar
+    extend self
+    using StringUtility
+
+    # Gets the total number of contributions for the past year.
+    # @param user [String] The username to get this data for.
+    # @return [Int] An integer value of their total contributions this year.
+    # @return [Nil] If the user cannot be found.
+    def get_total_year(user)
+      source = get_page_source(user)
+      return nil if source.nil?
```

Comment:
Would be better to either throw a custom NoUserFound error or to wrap this up in something